org.learningequality.Kolibri.yaml

```diff
@@ -106,10 +106,12 @@ modules:
       - >
         rm -rf
         /app/lib/python3.7/site-packages/kolibri/dist/cext/cp27
-        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp34
-        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp35
-        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp36
+        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp34/Windows
+        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp35/Windows
+        /app/lib/python3.7/site-packages/kolibri/dist/cext/cp36/Windows
```

Comment:
I think the only one we need to keep here is `cp34/Linux`, at least for x86/x64 systems.
![image](https://user-images.githubusercontent.com/618416/91348143-3a74cd80-e798-11ea-902c-5448a5625e89.png)
(the lowercase "linux" directories look to be just for ARM -- though I guess sometimes a Flatpak might be installed on an ARM system as well, so perhaps keeping those is best, I guess?)